Reference TKI and NHI in docs (#1840)
Deltares/Ribasim: docs/index.qmd

on top of the [SciML: Open Source Software for Scientific Machine Learning](https://sciml.ai/)
libraries.

> The first version of Ribasim is developed by [Deltares](https://www.deltares.nl) as part of a consortium for the Dutch watersystem.

Reviewer: ```suggestion
The initial version of Ribasim is developed by [Deltares](https://www.deltares.nl) as part of a consortium for the Dutch watersystem.
```

> This activity is co-funded by TKI Deltatechnology from the Dutch PPS innovation programme grant from the Ministry of Economic Affairs.

Author: ```suggestion
This activity is co-funded by [TKI Deltatechnology](https://tkideltatechnologie.nl/), a Dutch public–private partnership innovation program from the Ministry of Economic Affairs.
```

> It will be used as the surface water module of the [Netherlands Hydrologic Instrument (NHI)](https://nhi.nu/)

Reviewer: ```suggestion
Ribasim will be used as the surface water module of the [Netherlands Hydrologic Instrument (NHI)](https://nhi.nu/).
```
